week-2/M-array-map/exercise.js

```diff
@@ -10,7 +10,7 @@ function double(num) {
 }
 
 var numbers = [1, 2, 3, 4];
-var numbersDoubled; // complete this statement (use map and the double function)
+var numbersDoubled = numbers.map(double); // complete this statement (use map and the double function)
```

Comment:
`double` takes one parameter. Can you add it in? 
Here's some help: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map#Mapping_an_array_of_numbers_using_a_function_containing_an_argument

Reply:
Thanks

Reply:
Hi, I wanted to comment on work work as a part of my task, but it looks like Lucy is taking more than good care of you ;D enjoy reviewing your work on the light of her comments ;)

Reply:
@NajlaSoltana  thanks, sure @LucyMac has already reviewed my work, but you more than welcome to review it again.......